tests/Hamcrest/File/IsFileNamedTest.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * @author Vasek Brychta <vaclav@brychtovi.cz>
+ */
+
+namespace Hamcrest\File;
+
+use Hamcrest\AbstractMatcherTest;
+
+class IsFileNamedTest extends AbstractMatcherTest
+{
+	private static $FILE_PATH;
+	private static $FILE;
+	private static $ACTUAL_NAME;
+	private static $ACTUAL_NAME_START;
+
+	/**
+	 * @beforeClass
+	 */
+	public static function initializeFiles()
+	{
+		self::$FILE_PATH = __FILE__;
+		self::$FILE = new \SplFileInfo(self::$FILE_PATH);
+		self::$ACTUAL_NAME = pathinfo(self::$FILE_PATH, PATHINFO_BASENAME);
+		self::$ACTUAL_NAME_START = substr(self::$ACTUAL_NAME, 0, 5);
+	}
+
+	protected function createMatcher()
+	{
+		return IsFileNamed::aFileNamed(not(anything()));
```

Comment:
Something isn't right here. Should the `createMatcher` method have an argument given to the matcher and then all tests in this class create matcher via the `createMatcher` method?

Please see how it's implemented in other matcher tests, that have arguments.